src/lib/get-last-tag.ts

```diff
@@ -2,5 +2,5 @@
 import cmd from './cmd';
 
 export async function getLastTag(): Promise<string> {
-  return await cmd('git describe --tags --abbrev=0');
+  return await cmd('git fetch --prune --unshallow || true; git describe --tags --abbrev=0');
```

Comment:
just curious, whats the rationale of the `|| true`? shouldn't git fetch work in all envs? what error/s are we trying to swallow?

Reply:
`|| true` guarantees a successful exit. Just in case, extra careful.

Reply:
Yep :) I understand what it does (happy exit regardless/swallow errors),
I'm asking why we added it (eg we hit error x in testing or think that y could happen, in which case failure of this command is fine and we want to ignore it and still run the second thing).

I guess it comes to, do you feel it be more confusing or helpful if someone's hitting this `|| true` path in the future?
You understand this code better than I do, so happy to go with your advice here of course :)